packages/tasks/src/model-libraries.ts

```diff
@@ -1170,6 +1170,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: `path_extension:"pt"`,
 		snippets: snippets.vui,
 	},
+	"wan2.2": { 
+		prettyLabel: "Wan2.2",
+		repoName: "Wan2.2",
+		repoUrl: "https://github.com/Wan-Video/Wan2.2",
+		countDownloads: `path:"high_noise_model/config.json" OR path:"low_noise_model/config.json"`,
```

Comment:
```suggestion
		countDownloads: `path_filename:"config" AND path_extension:"json"`,
```

An even better rule is to set it to "filename is config and extension is json" => this way all config.json files in all subdirectories are counted correctly. Meaning it'll work with:
- https://huggingface.co/Wan-AI/Wan2.2-TI2V-5B/tree/main
- https://huggingface.co/Wan-AI/Wan2.2-T2V-A14B/tree/main
- https://huggingface.co/Wan-AI/Wan2.2-I2V-A14B/tree/main